Juris.js

```diff
@@ -108,10 +108,14 @@ function scrapeArticle(doc, url) {
 	item.title = ZU.trimInternal(myTitle);
 	// sometimes the title contains the authors at the end after separate <br/> tags
 	// in this case, the div is structured differently
-	// example article: BB 2014, 3
+	// example article: BB 2014, 3; CR 2016, 28; WRP 2016, 420
 	var temp = ZU.xpath(doc, "//div[@class='docLayoutTitel']/h3/following-sibling::br");
-	if (temp.length>0) {	// yes, there is additional information, then grab from another source
-		item.title = ZU.xpathText(doc, "//div[@class='docLayoutTitel']/h3/a");
+	if (temp.length>0) {	// yes, there is additional information, then grab from another source (which unfortunately differs sometimes)
+		var temp2 = ZU.xpathText(doc, "//div[@class='docLayoutTitel']/h3/a");
```

Comment:
When `a` is present, is there something we don't want from the `h3` that would not be inside the `a`? I.e. can we not just use the `xpathText` below to begin with?

If no, then use
```
var temp2 = ZU.xpathText(doc, "...")
    || ZU.xpathText(doc, "...");
```
which is more concise.